Homepage implementation

DeanCivlin opened the pull request:
* This Commit and PR are based on PR #94.
 * Implemented Navbar as part of the App Layout
 * Implemented home page which extends App Layout
 * Added new Transparent logo
 * updated urls.py and views.py due to homepage implementation

DeanCivlin commented:
@razashtar93 @Omeramsc @Yarboa @bregman-arie 
Would you please take a look at this PR?
It extends @razashtar93 PR #94

razashtar93 reviewed (approved):
Good job Dean!

Yarboa commented on templates/landing/homepage.html:
```diff
@@ -0,0 +1,62 @@
+{% load static %}
+{% block content %}
+<!DOCTYPE html>
+<html lang="en">
```
I do not understand

There is html and body landing/app_layout.html'
Am i wrong here?

DeanCivlin replied:
@Yarboa, 
If I understand your question correctly, Yes there are two HTML files.
1. `app_layout.html` that will wrap each of the platform pages
 (contains only the navbar and main container for the wrapped content
2. `homepage.html` which extends the `app_layout.html` and contains the home page content.

LMK if you meant otherwise

Yarboa replied:
so why do you need two heads.
MHO one could organize the main menu and page format,
Once calling to new html page, only the relevant body content is changing with bootstrap components, is there some thing prevent you from doing this?

DeanCivlin replied:
@Yarboa done

Yarboa commented on templates/landing/homepage.html:
```diff
@@ -0,0 +1,62 @@
+          <h3 class="my-2">Choose University</h3>
+          <div>
+            <select aria-label="choose-university" id="selectContent">
+              <option selected>Choose University</option>
```
this landing page is submitted 3 times
We would expect to have dynamic option. 
Parse a list of universities and run for loop in tags,
In case you want to add Universities to the list, add in migration test files

Please review the the UI talk. of the blog post

DeanCivlin replied:
@Yarboa,
Me and @razashtar93 are working together to make the lust dynamic and sourced fro the DB

Yarboa replied:
Is it in this PR? or another different one? 
In case it is different please note it in PR description

DanielItkes replied:
@Yarboa 
From what I underatand from the PR description, Dean is pushing this PR on the branch of Raz's PR that is already merged.

Yarboa replied:
yes, he could rebase, and see that this was not implemented,
In order not to miss it now,
Please rebase and fix
see this link
https://github.com/redhat-beyond/AcadeMe/pull/94/files#r767901359

DeanCivlin replied:
@Yarboa done

Yarboa commented on templates/landing/app_layout.html:
```diff
@@ -0,0 +1,41 @@
+{% load static %}
+<!DOCTYPE html>
+<html lang="en">
```
In case it s a layout for all the html files
it should be in higher hierarchy for other tamplates 

── templates
    └── app_layout.html
   ├── landing
   │   └──  homepage.html
   └── registration
       └── login.html

DanWaldman reviewed (approved):
LGTM

DanielItkes reviewed (approved):
Nice, good work.

Yarboa commented:
Please rebase

DeanCivlin commented:
@Yarboa @Omeramsc @bregman-arie 

1. I rebased and merged the `main` code to my branch.

2. In addition, I removed redundant HTML tags from homepage.html

Please take a look

Yarboa reviewed (approved):
LGTM

DeanCivlin commented:
@Yarboa Thanks for Approving the PR! would you please merge the code to `main`?
